CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 18 end 2015-01-26
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+18+Ending+Jan+26%2C+2015%22+label%3Aenhancement)
+
+- Providers
+  - Added refresh status and errors, viewable on Provider Summary Page
+  - OpenStack Host collection and metrics
+  - Amazon EC2:  Added C4 instance types
+- Rest API: VM start, stop, suspend, delete actions added.
+- UI
+ - Multi-character set language support
+ - Cloud Stacks summary and list views
+- Service Dialogs: Dynamic field support for text boxes and text area boxes
+- IPv6:  Pass IPv6 safe address to the socket layer
+
+
+### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+18+Ending+Jan+26%2C+2015%22+label%3Aenhancement)
+
+- Rails Updates
+  - Moved to Rails 4 finders
+  - Removed patches against the logger classes
+  - Removed assumptions that associations are eagerly loaded
+  - Got the app running on Rails 4 (webpages are usable, but tests still fail)
+- Fog gem updates for OpenStack
+
+### [Removed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+18+Ending+Jan+26%2C+2015%22+label%3A%22technical+debt%22)
```

Comment:
No need for a removed section is nothing was removed...I admit I did not actually check if anything needs to be put in this section yet...so you may want to keep it around.